lib/entity.js

```diff
@@ -10,16 +9,39 @@ function Entity (entity) {
   this.type = entity.type || 'Unknown';
   this.name = entity.name || this.id;
 
-  this.experience = 0;
+  this.EXP = 0;
```

Comment:
Avoid changing the name of properties on templates, since other parts of the program most likely depend on them.  Also, as a general rule, try to keep all properties human-friendly — there's no need to shorten variable names in modern languages!

Reply:
Got it, I'll change it back to `this.experience`